src/blockchains/erc20/lib/fetch_events.ts

```diff
@@ -37,7 +37,7 @@ export function decodeEventBasicInfo(event: any, timestampsCache: TimestampsCach
 /**Transfer(address,address,uint256)
  * Used by all ERC20 tokens
  **/
-function decodeTransferEvent(web3Wrapper: Web3Interface, event: any, timestampsCache: TimestampsCacheInterface): ERC20Transfer | null {
+function decodeTransferEvent(event: any, timestampsCache: TimestampsCacheInterface): ERC20Transfer | null {
```

Comment:
There are a lot of this type of changes. Argument is no longer provided as the function can use static instance.